sesamutils/sesamlogger.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import os
+
+
+def sesam_logger(logger_name, timestamp=False):
+    logger = logging.getLogger(logger_name)
+    level = None
+    if os.getenv('LOG_LEVEL') is not None:
```

Comment:
Since I know you like performance, can you please compare the speed when reading an environment variable multiple times vs. assigning to variable and using that?

**Another note:**
`if os.getenv('LOG_LEVEL') is not None:`
is the same as
`if os.getenv('LOG_LEVEL'):`

Reply:
fixed :-)

Reply:
did you do a comparison? I'm curious!